modules/ROOT/pages/runtime-manager-switch-env.adoc

```diff
@@ -1,19 +1,35 @@
-= Switch Environments Using Runtime Manager
+= Using Different Environments in Runtime Manager
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform PCE"]
+image:logo-rtf-active.png[link="/runtime-fabric/deploy-to-runtime-fabric", title="Runtime Fabric"]
```

Comment:
Question: is there a reason why we're not linking the Runtime Fabric icon to /deployment-strategies/

(This would apply to all pages, not just this one).

Reply:
Good question! I will change it.